weave/tests/test_basket.py

```diff
@@ -498,38 +498,6 @@ def test_basket_init_fails_if_uuid_does_not_exist(test_pantry):
         )
 
 
-def test_basket_pantry_name_does_not_exist(test_pantry):
```

Comment:
This was failing, but it felt like a duplicate test anyway. Similar to test_pantry_fails_with_bad_path that I had to change with this update.